server/mldb_server.h

```diff
@@ -142,6 +143,28 @@ struct MldbServer: public ServicePeer, public EventRecorder {
     Utf8String prefixUrl(Utf8String url) const;
     std::string prefixUrl(std::string url) const;
     std::string prefixUrl(const char* url) const;
+    
+    InProcessRestConnection mldbPerform(
```

Comment:
Also, I don't think it's ambiguous: it's a REST server, so get is pretty clear.  But if we have a prefix, it should be rest.